deploy/helm/sumologic/values.yaml

```diff
@@ -4061,7 +4065,19 @@ metadata:
               - batch
             exporters:
               - sumologic/systemd
-
+          logs/otc/containers:
```

Comment:
Can you update the pipelines in vagrant's values.yaml as well so that they are not "duplicated" in vagrant env?